release-notes/current.yaml

```diff
@@ -14,6 +14,7 @@ new features: |
   Allow matchExpressions in TargetSelector
   Add defaulter for gateway-api resources loading from file to be able to set default values.
   Added support for defining Lua EnvoyExtensionPolicies
+  Added support for HorizontalPodAutoscaler to helm chart
```

Comment:
I assume this was just missed? I don't see any extra changes required here

Reply:
my bad, misreading.